platform/android/java/editor/src/main/java/org/godotengine/editor/BaseGodotEditor.kt

```diff
@@ -81,12 +81,13 @@ open class GodotEditor : GodotActivity() {
 		// Command line arguments
 		private const val FULLSCREEN_ARG = "--fullscreen"
 		private const val FULLSCREEN_ARG_SHORT = "-f"
-		private const val EDITOR_ARG = "--editor"
-		private const val EDITOR_ARG_SHORT = "-e"
-		private const val EDITOR_PROJECT_MANAGER_ARG = "--project-manager"
-		private const val EDITOR_PROJECT_MANAGER_ARG_SHORT = "-p"
-		private const val BREAKPOINTS_ARG = "--breakpoints"
-		private const val BREAKPOINTS_ARG_SHORT = "-b"
+		internal const val EDITOR_ARG = "--editor"
```

Comment:
Not familiar with kotlin, what's the difference between `private` and `internal`, and should the other two args also be `internal` (`--fullscreen`/`-f`)?

Reply:
-`private` is the same as Java while `internal` scope allows access to components within the same module (e.g: `lib` module, `editor` module, `app` module).

The `fullscreen` arguments are only used within this class so following the pattern of limited access, I went with the `private` scope for them. The other arguments are used by classes other than `BaseGodotEditor`.